app/assets/stylesheets/frontend/print/_document.scss

```diff
@@ -52,31 +52,62 @@ article {
 .govspeak {
   h2 {
     clear: both;
-    padding-top: $gutter*2;
+  }
+  figure {
+    margin: 0;
   }
   .attachment {
     clear: both;
-    .attachment-thumb,
-    .attachment-details {
-      float: left;
-      width: 15%;
-    }
     .attachment-thumb {
-      margin: 0 $gutter*2 $gutter*2 0;
-      a {
-        img {
-          width: 100%;
-        }
-        &:after {
-          content: "";
-        }
-      }
+      display: none;
     }
     .attachment-details {
-      width: 80%;
       h3 {
         margin-top: 0;
       }
+      .accessibility-warning {
+        h2 {
+          @include core-14;
+          padding: 0;
+          margin: 0;
+        }
+        .toggler {
+          display: none;
+        }
+      }
+    }
+  }
+
+  table {
+    border: 1pt solid rgb(125,125,125);
```

Comment:
Cool. I think we should try use variables where possible for consistency. Otherwise, I'm happy with this pull request.